runtime/doc/health.txt

```diff
@@ -21,6 +21,23 @@ To run all healthchecks, use: >vim
 <
 Plugin authors are encouraged to write new healthchecks. |health-dev|
 
+                                                                    *g:health*
+g:health  This global variable controls the behavior and appearance of the
+          `health` floating window. It should be a dictionary containing the
+          following optional keys:
+          - `style`: string? Determines the display style of the `health` window.
+                            Set to `'float'` to enable a floating window. Other
+                            styles are not currently supported.
+          - `syntax`: string? Specifies the syntax highlighting to use
+                            for the content of the `health` window.
+                            Defaults to an empty string if not provided.
+          - `border`: (string|table) Configures the border style of the floating window.
```

Comment:
Don't need this if we get https://github.com/neovim/neovim/pull/31074 , right?

Reply:
yes